src/main/java/com/feywild/feywild/FeywildMod.java

```diff
@@ -131,6 +134,18 @@ protected void initRegistration(RegistrationBuilder builder) {
     @Override
     protected void setup(FMLCommonSetupEvent event) {
         event.enqueueWork(() -> {
+
+            AxeItem.STRIPPABLES = new ImmutableMap.Builder<Block, Block>().putAll(AxeItem.STRIPPABLES)
```

Comment:
I did not test it but quickly checking the forge source code, I thnk you can achieve this by overriding `getToolModifiedState` in `Block` and return the stripped block if the tool action is `AXE_STRIP`